From: Marco Felsch <m.felsch@pengutronix.de>
Date: Tue, 26 Mar 2019 09:50:55 +0100
Subject: [PATCH] ARM: dts: imx6qdl-skov: fix touchscreen orientation

Report the correct values to the user space so libinput must not do that
inversion.

Drop the min/max values to ensure the correct behaviour in case of using
libinput. The min/max values aren't validated by the tslib but are
validated by libinput. We don't need to specify these values at all since
the raw values gets mapped to the screen size during the calibration.

Signed-off-by: Marco Felsch <m.felsch@pengutronix.de>
---
 arch/arm/boot/dts/imx6qdl-skov-cpu.dtsi | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-skov-cpu.dtsi b/arch/arm/boot/dts/imx6qdl-skov-cpu.dtsi
index 0ff40206a283..5a57d4345469 100644
--- a/arch/arm/boot/dts/imx6qdl-skov-cpu.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-skov-cpu.dtsi
@@ -340,10 +340,6 @@
 		spi-max-frequency = <1000000>;
 		vcc-supply  = <&reg_3p3v>;
 
-		ti,x-min = /bits/ 16 <800>;
-		ti,x-max = /bits/ 16 <3400>;
-		ti,y-min = /bits/ 16 <500>;
-		ti,y-max = /bits/ 16 <3400>;
 		ti,x-plate-ohms = /bits/ 16 <642>;
 		ti,y-plate-ohms = /bits/ 16 <295>;
 		ti,pressure-max = /bits/ 16 <1500>;
@@ -352,6 +348,9 @@
 		ti,debounce-tol = /bits/ 16 <(~0)>;
 		ti,debounce-rep = /bits/ 16 <8>;
 
+		touchscreen-swapped-x-y;
+		touchscreen-inverted-y;
+
 		linux,wakeup;
 	};
 };
